docs/create-modify-element.mdx

```diff
@@ -229,6 +233,8 @@ outerEle.appendChild(fragment)
 // This will remove all content from fragment and 
 // insert them within outerEle.
 
+// .appendChild()/.insertBefore() - can be used to move the whole content of the fragment to the actual DOM.
```

Comment:
Let's add this as note (within details) in appendChild section?? Then, we can link back to that note here.

Reply:
Sorry didn't get this. Where exactly line 236 content should be put in appendChild ? And also what should be done for insertBefore ? How do you plan to link back to that note ?

Reply:
Let me make this change, it'll be easier to show that way. (i meant #abc style link within same page, if we can somehow add a anchor within the note)

Reply:
Cool 👍